### PR TITLE
Update form submissions API URL

### PR DESCRIPTION
**Pull request by esoergel**
We just added a new api-specific endpoint for form submissions:
https://github.com/dimagi/commcare-hq/pull/30784
https://confluence.dimagi.com/display/commcarepublic/Submission+API

### CODE REVIEW

**Comment by esoergel** [comment]
@kaapstorm @orangejenny

**Comment by kaapstorm** [comment]
Thanks for this @esoergel !

**Review by kaapstorm — APPROVED** [review]

**Review by orangejenny — APPROVED** [review]

**Comment by esoergel** [comment]
@kaapstorm do you have write access to merge this?  I'm waiting on [this ticket](https://dimagi-dev.atlassian.net/browse/SAAS-12859) before I can merge.

**Pull request merged**
